src/types/field/fieldTypes.ts

```diff
@@ -52,3 +52,9 @@ export enum MyInfoAttribute {
   WorkpassExpiryDate = 'workpassexpirydate',
   GraduationYear = 'gradyear',
 }
+
+export enum SPCPValidatedFields {
```

Comment:
do you think it might be better to name this something like `SPCPFieldTitle`? `SPCPValidatedFields` may be a little vague as to which part of the field it's referring to (title, answer, field type, description etc)

Reply:
thanks! edited